source/mainnet/net/resources/glossary.rst

```diff
@@ -398,6 +412,13 @@ Passive delegation
 
 A form of delegation where a delegator's stake is effectively distributed among all baker pools. It is not associated with a specific baker. Delegators earn lower rewards when delegating to passive delegation than when delegating to a specific baker pool. However, passive delegation is not affected by poor performance of a single baker.
 
+.. _glossary-range-proofs:
+
+Range proofs
+============
+
+A range proof asks a user to prove that they meet an attribute within a range of values. For example, when renting a car, you might need to prove that you are between 25 and 65 to the car rental company. This could be constructed as a range proof.
```

Comment:
```suggestion
A range proof asks a user to prove that they meet an attribute within a range of values. For example, when renting a car, you might need to prove that you are between 25 and 65 years old to the car rental company. This could be constructed as a range proof.
```